contracts/src/v0.8/llo-feeds/dev/FeeManager.sol

```diff
@@ -350,7 +359,17 @@ contract FeeManager is IFeeManager, ConfirmedOwner, TypeAndVersionInterface {
    * @notice Gets the current version of the report that is encoded as the last two bytes of the feed
    * @param feedId feed id to get the report version for
    */
-  function getReportVersion(bytes32 feedId) internal pure returns (bytes32) {
+  function _getReportVersion(bytes32 feedId) internal pure returns (bytes32) {
     return REPORT_VERSION_MASK & feedId;
   }
+
+  /// @inheritdoc IFeeManager
+  function payLinkDeficit(bytes32 configDigest) external onlyOwner {
+    uint256 deficit = s_linkDeficit[configDigest];
+
+    delete s_linkDeficit[configDigest];
+    i_rewardManager.onFeePaid(configDigest, address(this), deficit);
```

Comment:
Do you need a check on the current balance before doing this? onFeePaid seems to use transferFrom and not safeTransferFrom, I'd also recommend changing that.

Reply:
They ultimately do the same thing for ERC20 tokens except safeTransferFrom uses more gas. transferFrom() will revert if the transfer fails.

Reply:
It's always linked or wrapped native in the current implementation